toxcore/state.c

```diff
@@ -103,3 +103,21 @@ void lendian_to_host32(uint32_t *dest, const uint8_t *lendian)
 #endif
     *dest = d;
 }
+
+void host_to_lendian_bytes16(uint8_t *dest, uint16_t num)
+{
+#ifdef WORDS_BIGENDIAN
```

Comment:
At some point we should stop depending on these and write portable code. Do you want to do that now or defer it?

Reply:
Hmm... I wouldn't know how best to do it, and I understand we intend to redo saving completely eventually. So I'd rather defer.